<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="30" started-at="2016-08-11T11:23:20Z" finished-at="2016-08-11T11:23:20Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="30" started-at="2016-08-11T11:23:20Z" finished-at="2016-08-11T11:23:20Z">
      <class name="com.test.sample.sampleTest">
        <test-method status="PASS" signature="beforeTest()[pri:0, instance:com.test.sample.sampleTest@553f17c]" name="beforeTest" is-config="true" duration-ms="0" started-at="2016-08-11T16:53:20Z" finished-at="2016-08-11T16:53:20Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeTest -->
        <test-method status="FAIL" signature="test()[pri:0, instance:com.test.sample.sampleTest@553f17c]" name="test" duration-ms="10" started-at="2016-08-11T16:53:20Z" finished-at="2016-08-11T16:53:20Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at com.test.restAPIUtitlities.RequestBuilder.getrequestSpecHeaders(RequestBuilder.java:21)
	at com.test.sample.sampleTest.test(sampleTest.java:22)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:137)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:58)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test -->
        <test-method status="PASS" signature="afterTest()[pri:0, instance:com.test.sample.sampleTest@553f17c]" name="afterTest" is-config="true" duration-ms="0" started-at="2016-08-11T16:53:20Z" finished-at="2016-08-11T16:53:20Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterTest -->
      </class> <!-- com.test.sample.sampleTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
